plugin/src/sbt-test/scalafmt-sbt/sbt/build.sbt

```diff
@@ -147,22 +158,22 @@ TaskKey[Unit]("check") := {
   assertContentsEqual(
     file(s"p7/src/main/scala/Test.scala"),
     """
-      |object Test {
-      |  foo(
-      |    a, // comment
-      |    b
-      |  )
+      |object
+      |Test
+      |{
+      |  foo(a, // comment
+      |    b)
       |}
     """.stripMargin
   )
   assertContentsEqual(
     file(s"p7/src/test/scala/MainTest.scala"),
     """
-      |object MainTest {
-      |  foo(
-      |    a, // comment
-      |    b
-      |  )
+      |object
+      |MainTest
+      |{
+      |  foo(a, // comment
+      |    b)
```

Comment:
Previously, if `.scalafmt.conf` is missing, the default configuration is used for running scalafmt.
As of this commit, sbt-scalafmt does nothing if the configuration is missing.

Reply:
Good to test this 👍